settings_test.py

```diff
@@ -13,8 +13,15 @@
 
 DEMO_UPLOADS_ROOT = '/home/vagrant/uploads/demos'
 
-LOGGING['loggers']['django.db.backends.schema'] = {
-    'handlers': ['console'],
-    'propagate': True,
-    'level': 'WARNING',
-}
+LOGGING['loggers'].update({
+    'django.db.backends.schema': {
```

Comment:
nit: I recently updated my `settings_test.py` to `django.db.backends` and not just `schema` because some other DB logs were cluttering my test output.